tests/fixtures.py

```diff
@@ -26,14 +29,14 @@ def testdb_uri(db_backend):
         # This is designed to work on Travis CI
         return 'postgresql://postgres@localhost:5432/hydra_base_test'
     elif db_backend == 'mysql':
-        return 'mysql+mysqldb://root@localhost/hydra_base_test'
+        return 'mysql+mysqldb://root:root@localhost/hydra_base_test'
     else:
         raise ValueError('Database backend "{}" not supported when running the tests.'.format(db_backend))
 
 
 @pytest.fixture(scope='function')
 def engine(testdb_uri):
-    engine = create_engine(testdb_uri)
+    engine = create_engine(testdb_uri, encoding='utf-8')
```

Comment:
I think this is default? But perhaps not in Python 2?

Reply:
possibly yes but this makes it explicit, so let's leave it.